config/config.exs

```diff
@@ -14,6 +14,10 @@ config :logger, :console,
   format: "$time $metadata[$level] $message\n",
   metadata: [:request_id]
 
+config :course_planner, CoursePlanner.Mailer.Main,
+  adapter: Swoosh.Adapters.SMTP,
+  api_key: "some api key should be added here"
+
```

Comment:
You can use `adapter: Swoosh.Adapters.Local` to check sent e-mails in `/dev/mailbox` and put the SMTP config in `prod.exs`.